test/OrchardCore.Tests/Apis/Context/SiteStartup.cs

```diff
@@ -17,10 +18,15 @@ namespace OrchardCore.Tests.Apis.Context
     public class SiteStartup
     {
         public static ConcurrentDictionary<string, PermissionsContext> PermissionsContexts;
+        private static Mock<IHttpContextAccessor> mockHttpContextAccessor;
```

Comment:
Why does this need to be a static field instead of just constructing the mock in `ConfigureServices`?